src/System Application/App/AI/src/Azure OpenAI/AOAIOperationResponse.Codeunit.al

```diff
@@ -20,6 +20,7 @@ codeunit 7770 "AOAI Operation Response"
         Success: Boolean;
```

Comment:
These changes are part of https://github.com/microsoft/BCApps/pull/719

Reply:
Yes, but how should i separate those?

Reply:
We process it and then you can remove it.